### DISCUSSION TITLE
404 Error - Install not found

### DISCUSSION THREAD

**Discussion by Firav** [Chevereto/Chevereto-Free#174] [Help needed]
Heyo y'all, I've been recently attempting to install chevereto-free on a Debian 10 environment.
I've followed whats stated in https://chevereto-free.github.io/setup/server/requirements.html, but I still cant seem to pull up the install.

Heres the configuration of the virtual host:
```
<VirtualHost *:80>
    ServerAdmin ***
    DocumentRoot "/var/www/img"
    ServerName ***

<Directory /var/www/img>
        Options -Indexes +FollowSymLinks +MultiViews
        AllowOverride All
	Require all granted
</Directory>

<Directory /var/www/img/images>
    AllowOverride None
    <FilesMatch "\.(?:[Pp][Hh][Pp][345]?|[Pp][Hh][Tt][Mm][Ll])|(po|sql|html?)$">
        <IfModule !mod_authz_core.c>
            Order Allow,Deny
            Deny from all
        </IfModule>
        <IfModule mod_authz_core.c>
            Require all denied
        </IfModule>
    </FilesMatch>
    <IfModule mod_php7.c>
        php_flag engine off
    </IfModule>
    <FilesMatch ".+\.*$">
        SetHandler !
    </FilesMatch>
    <IfModule mod_rewrite.c>
        RewriteRule ^.*\.php$ - [F,L]
    </IfModule>
</Directory>
</VirtualHost>
```

Within the apache2.conf, its configured as so:
```
<Directory /var/www/>
	Options Indexes FollowSymLinks
	AllowOverride All
	Require all granted
</Directory>
```

What am I missing? I've referred to #7 in regards to this issue also, but cant seem to reach a solution.

**Comment by rodber** [comment]
You took the examples and chop `www` form the path, so your paths must reflect your config. I ignore the context of your server, but you should refer to your systems logs to understand why your edits aren't working.
